### PR TITLE
Updated README and added CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by harmut01**
## Summary of changes
Created CONTRIBUTING.md
Updated license information in README.md

## Impact of changes

None
## Migration actions required

None.
## Documentation

None.
## Pull request type

[x] Patch update (Bug fix / Target update / Docs update / Test update / Refactor)
[] Feature update (New feature / Functionality change / New API)
[] Major update (Breaking change E.g. Return code change / API behaviour change)

## Test results

[X] No Tests required for this change (E.g docs only update)
[] Covered by existing mbed-os tests (Greentea or Unittest)
[] Tests / results supplied as part of this PR
## Reviewers
@ARMmbed/mbed-os-core

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,6 @@ If you have problems, you can review the [documentation](https://os.mbed.com/doc
 
 ### License and contributions
 
-The software is provided under the Apache-2.0 license. Contributions to this project are accepted under the same license. Please see contributing.md for more information.
+The software is provided under the Apache-2.0 license. Contributions to this project are accepted under the same license. Please see [contributing.md](./CONTRIBUTING.md) for more information.
 
 This project contains code from other projects. The original license text is included in those source files. They must comply with our license guide.
```
Comment (LDong-Arm) [inline README.md:146]:
```suggestion
This project contains code from other projects. The original license text is included in those source files. They must comply with our [license guide](https://os.mbed.com/docs/mbed-os/latest/contributing/license.html)
```

**Review by evedon — CHANGES_REQUESTED** [review]
Please also add the license file

**Review by LDong-Arm — CHANGES_REQUESTED** [review]
~Need to add the actual LICENSE file~ Oh duplicated comment

**Review by evedon — APPROVED** [review]

**Comment by evedon** [comment]
@LDong-Arm Your comment was addressed so are you happy to approve?

**Review by LDong-Arm — APPROVED** [review]

**Comment by evedon** [comment]
@harmut01 Could you rebase your PR as this should remove the first commit?

**Comment by harmut01** [comment]
@evedon I have rebased the PR, this should address your comment.

**Pull request merged**